openmetadata-ui/src/main/resources/ui/src/axiosAPIs/alertsAPI.ts

```diff
@@ -40,7 +40,20 @@ export const getAlertsFromId = async (
     params: {
       ...params,
       include: 'all',
-      fields: 'alertActions',
+    },
+  });
+
+  return response.data;
+};
+
+export const getAlertsFromName = async (
+  name: string,
+  params?: Pick<ListAlertsRequestParams, 'include'>
+) => {
+  const response = await axiosClient.get<Alerts>(`${BASE_URL}/name/${name}`, {
+    params: {
+      ...params,
+      include: 'all',
```

Comment:
@chirag-madlani , here in params we are expecting `include` value, but when passing the params we are overriding it with `include: 'all',`.

Can you confirm what we are trying to achieve here?

Reply:
We are expecting all as we are not having softDeleted in case of alerts so it's fine